components/MapView.js

```diff
@@ -399,6 +399,10 @@ class MapView extends React.Component {
     this._runCommand('fitToSuppliedMarkers', [markers, animated]);
   }
 
+  fitToCoordinates(coordinates, edgePadding, animated) {
```

Comment:
@spikebrehm I initially had `{ top: 0, right: 0, bottom: 0, left: 0 }` as a default argument, but then realized `setVisibleMapRect` could be called without `edgePadding` altogether. So I moved the conditional to the native side.

I don't mind switching to the default argument instead, whatever you guys prefer -- I'm easy.